tillerinobot/src/main/java/tillerino/tillerinobot/lang/Italiano.java

```diff
@@ -120,11 +120,11 @@ public GameChatResponse hug(OsuApiUser apiUser) {
  
 	@Override
 	public String help() {
-		return "Ciao! Sono il robot che ha ucciso Tillerino e ha preso possesso del suo account. Scherzo, ma uso lo stesso il suo account."
+		return "Ciao! Sono il robot che ha ucciso Tillerino e preso possesso del suo account. Sto scherzando, ma uso lo stesso il suo account."
 				+ " [https://twitter.com/Tillerinobot stato e aggiornamenti]"
 				+ " - [https://github.com/Tillerino/Tillerinobot/wiki comandi]"
-			        + " - [http://ppaddict.tillerino.org/ ppaddict]"
-			        + " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact contatti]";
+			    + " - [http://ppaddict.tillerino.org/ ppaddict]"
+			    + " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact contatti]";
```

Comment:
bit nitpicky maybe, but in the interest of not introducing diffs where not needed: could you revert those whitespace changes in these 2 lines?